File: exercises/D111/solution.py
w, h = map(float, input().split())
bmi = w / (h * h)
print(f'{bmi:.3f}')
if bmi < 18.5:
    print('Underweight')
elif bmi < 23.0:
    print('Normal')
elif bmi < 25.0:
    print('Overweight')
else:
    print('Obese')

File: exercises/D111/solution.java
import java.util.*;

class D111 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        double w = in.nextDouble();
        double h = in.nextDouble();
        double bmi = w / (h * h);
        System.out.printf("%.3f\n", bmi);
        if (bmi < 18.5) {
            System.out.println("Underweight");
        }
        else if (bmi < 23.0) {
            System.out.println("Normal");
        }
        else if (bmi < 25.0) {
            System.out.println("Overweight");
        }
        else {
            System.out.println("Obese");
        }
    }
}
